base/file_name_utils.cpp

```diff
@@ -32,19 +31,17 @@ void GetNameFromFullPath(string & name)
     name = name.substr(i+1);
 }
 
-std::string GetNameFromFullPath(std::string const & path)
+std::string FileNameFromFullPath(std::string path)
```

Comment:
Do we need `std::` namespace here?

Reply:
Well, there is a _using std_ in the beginning, but I didn't change notation here. We are on the way to avoid using std in code.

Reply:
@strump `using namespace std;` in global namespace was ok in .cpp files. But after we started using [unity builds](https://en.wikipedia.org/wiki/Unity_build) for faster compilation, it leads to compilation errors in some cases.